imagick_class.c

```diff
@@ -3085,8 +3088,8 @@ PHP_METHOD(imagick, queryfontmetrics)
 			query_multiline = 0;
 		}
 	} else {
-		convert_to_boolean(multiline);
-		query_multiline = Z_BVAL_P(multiline);
+		convert_to_double(multiline);
+		query_multiline = Z_DVAL_P(multiline);
 	}
```

Comment:
I've just realised this is wrong - it should have been converted with `if (Z_TYPE_P(item) == IS_TRUE)` apparently.